veg2hab/io/common.py

```diff
@@ -57,16 +57,20 @@ class ShapefileInputs(BaseModel):
     vegtype_col_format: Literal["single", "multi"] = Field(
         description='"single" als complexen in 1 kolom zitten of "multi" als er meerdere kolommen zijn',
     )
-    sbb_of_vvn: Literal["SBB", "VvN", "beide"] = Field(
-        description='"VvN" als VvN de voorname vertaling is vanuit het lokale type, "SBB" voor SBB en "beide" als beide er zijn.'
+    welke_typologie: WelkeTypologie = Field(
+        description='Voornaamste typologie van waaruit de vertalingen worden uitgevoerd. ("SBB", "VvN", "rVvN", "SBB en VvN")',
```

Comment:
Deze moet ook nog aan AccessDBInputs toegevoegd worden toch?

Reply:
Oh, ik dacht dat dit altijd op basis van SBB was, dus de nieuwe opties zijn:
Digitale standaard: SBB of rVvN?
Shapefile: alle drie.

Reply:
Correct, behalve dat er bij shapefile 4 opties zijn, want je kunt ook SBB en VvN aanleveren :)